app/templating/schema_context.py

```diff
@@ -1,10 +1,10 @@
-from app import settings
 from app.utilities.date_utils import to_date
 
 
-def build_schema_context(metadata, aliases, answer_store, group_instance=0):
+def build_schema_context(metadata, aliases, answer_store, max_repeats, group_instance=0):
```

Comment:
Maybe worth setting a default for max_repeats as it's currently a limit we have to enforce? Tiny reduction in boiler plate for callers but every little helps.